packages/features/data-table/components/filters/AddFilterButton.tsx

```diff
@@ -62,7 +62,10 @@ function AddFilterButtonComponent<TData>(
             <CommandList>
               <CommandEmpty>{t("no_columns_found")}</CommandEmpty>
               {filterableColumns.map((column) => {
-                const isVisible = table.getColumn(column.id)?.getIsVisible();
+                const showHiddenIndicator =
```

Comment:
Do not show "hidden" indicator if the column is hidden by initialState (not by user interaction)